Components/toast-component/toast.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/Components/toast-component/toast.css">
+    <title>Toast</title>
+</head>
+<body>
+    <div class="toast">
+        <p>This is normal toast</p>
+    </div>
+    <br>
+    <div class="toast sucess">
```

Comment:
It would be much more readable if you add toast-success class instead of just success.